src/pdf/DriverOverview/DriverOverviewPdf.tsx

```diff
@@ -115,6 +116,9 @@ const styles = StyleSheet.create({
     instructionsColumnWidth: {
         width: "25%",
     },
+    numberOfLabelsColumnWidth: {
+        width: "15%",
```

Comment:
i think this being 20% and instructions also being 20% will work well as per my previous comment